Client/Frontend/Strings.swift

```diff
@@ -1668,6 +1668,45 @@ extension String {
                     tableName: "Settings",
                     value: "Default Search Engine",
                     comment: "Accessibility label for default search engine setting.")
+                public static let LearnAboutSuggestions = MZLocalizedString(
+                    key: "Settings.Search.Accessibility.LearnAboutSuggestions.v121",
+                    tableName: "Settings",
+                    value: "Learn more about Firefox Suggest",
+                    comment: "Accessibility label for Learn more about Firefox Suggest.")
+            }
+            
+            public struct Suggest {
+                public static let AddressBarSettingsTitle = MZLocalizedString(
+                    key: "Settings.Search.Suggest.AddressBarSetting.Title.v122",
+                    tableName: "Settings",
+                    value: "Address bar - Firefox Suggest",
```

Comment:
Whenever we use the app name in any string, we should be using a placeholder `%@`. The comment, additionally, should have a "Placeholder refers to app name". Firefox doesn't get translated. Then, in the place where you're using the string, you'd do something like 
```
.... = String.localizedStringWithFormat(.Settings.Search.Suggest.AddressBarSettingsTitle,
                                                                  AppName.shortName.rawValue)
```

Reply:
Ahh, in this particular case, I think we intentionally don't want to use the app name: on Desktop and Android, we treat "Firefox Suggest" as a brand string, similar to "Firefox Accounts" (before we renamed it to "Mozilla accounts" 😅). It would be incorrect for this string to expand to, e.g., "Firefox Beta Suggest", "Nightly Suggest", or "Focus Suggest".